docs/wiki/The library - Module usage.md

```diff
@@ -19,74 +15,164 @@ This section provides a guideline on how to use the CARML Bicep modules.
 
 This section shows you how to deploy a Bicep template.
 
-- [Deploy local template](#deploy-local-template)
-- [Deploy remote template](#deploy-remote-template)
+## PowerShell
 
-## Deploy local template
+This sub-section gives you an example on how to deploy a template from your local drive (file) or a publicly available remote location (URI).
 
-This sub-section gives you an example on how to deploy a template from your local drive.
-
-### **Local:** PowerShell
-
-This example targets a resource group level template.
+<details>
+<summary><i>Resource Group</i> scope</summary>
 
 ```PowerShell
 New-AzResourceGroup -Name 'ExampleGroup' -Location "Central US"
 
 $inputObject = @{
- DeploymentName    = 'ExampleDeployment'
- ResourceGroupName = 'ExampleGroup'
- TemplateFile      = "$home\ResourceModules\arm\Microsoft.KeyVault\vault\deploy.bicep"
+  DeploymentName    = 'ExampleDeployment'
+  ResourceGroupName = 'ExampleGroup'
+  # Using a local reference
+  TemplateFile      = "$home\ResourceModules\arm\Microsoft.KeyVault\vault\deploy.bicep"
+  # Using a remote reference
+  # TemplateUri     = 'https://raw.githubusercontent.com/Azure/ResourceModules/main/arm/Microsoft.KeyVault/vaults/deploy.bicep'
 }
 New-AzResourceGroupDeployment @inputObject
 ```
 
-### **Local:** Azure CLI
+For more information please refer to the official [Microsoft docs](https://docs.microsoft.com/en-us/powershell/module/az.resources/new-azresourcegroupdeployment).
 
-This example targets a resource group level template.
+</details>
 
-```bash
-az group create --name 'ExampleGroup' --location "Central US"
-$inputObject = @(
-    '--name',           'ExampleDeployment',
-    '--resource-group', 'ExampleGroup',
-    '--template-file',  "$home\ResourceModules\arm\Microsoft.KeyVault\vault\deploy.bicep",
-    '--parameters',     'storageAccountType=Standard_GRS',
-)
-az deployment group create @inputObject
+<details>
+<summary><i>Subscription</i> scope</summary>
```

Comment:
I'd suggest to be even more detailed and specify at the beginning of each scope paragraph how the corresponding bicep module targetscope should look like

Reply:
Added